index.html

```diff
@@ -86,25 +86,311 @@ <h3>Use-case #4: Avoiding "Hall of Mirrors"</h3>
+            data-link-for="CaptureActionEventInit"
+            data-dfn-for="CaptureActionEventInit"
+          >
+            <dt>
+              <dfn>action</dfn>
+            </dt>
+            <dd>
+              The {{CaptureAction}} to initialize the event with.
+            </dd>
+          </dl>
+        </section>
+      </section>
+    </section>
+    <section id="capturing-side-actions">
+      <h2>Capturing Side for Actions</h2>
+      <p>
+        Capturing applications can enumerate available [=capture actions=] that
+        are supported on the video track they have obtained, by using
+        {{MediaStreamTrack/getSupportedCaptureActions}}, and can trigger those
+        actions by using {{MediaStreamTrack/sendCaptureAction}}.</li>
+      </ol>
+      <section id="capture-handle-get-supported-actions">
+        <h3>Enumerating supported actions and triggering them</h3>
+        <p>
+          When a {{MediaStreamTrack}} is a video track derived from screen-capture
+          of a <a data-cite="SCREEN-CAPTURE#dfn-browser">browser</a>
+          <a data-cite="SCREEN-CAPTURE#dfn-display-surface">display surface</a>,
+          {{MediaStreamTrack/getSupportedCaptureActions}} returns the set of
+          available [=capture actions=], if any, supported by the captured
+          application associated with this video track.
+        </p>
+        <pre class="idl">
+          partial interface MediaStreamTrack {
+            sequence&lt;DOMString&gt; getSupportedCaptureActions();
+            Promise&lt;undefined&gt; sendCaptureAction(CaptureAction action);
+          };
+        </pre>
+        <dl data-link-for="MediaStreamTrack" data-dfn-for="MediaStreamTrack">
+          <dt>
+            <dfn>getSupportedCaptureActions</dfn>
+          </dt>
+          <dd>
+            <p>When this method is invoked, the user agent MUST return [=this=]'
+            {{MediaDevices/[[AvailableCaptureActions]]}} if defined, or `[]` if not defined.</p>
+          </dd>
+          <dt>
+            <dfn>sendCaptureAction</dfn>
+          </dt>
+          <dd>
+            <p>When this method is invoked, the user agent MUST run the following steps:</p>
+            <ol>
+              <li>
+                If the [=relevant global object=] of [=this=] does not have
+                [=transient activation=], return a promise [=rejected=] with
+                {{InvalidStateError}}.
+              </li>
+              <li>
+                [=Consume user activation=].
+              </li>
+              <li>
+                Let |action| be the method's first argument.
+              </li>
+              <li>
+                If |action| is not in [=this=]' {{MediaDevices/[[AvailableCaptureActions]]}},
+                return a promise [=rejected=] with {{NotFoundError}}.
+              </li>
+              <li>
+                Let |p| be a new promise.
+              </li>
+              <li>
+                Run the following steps in parallel:
```

Comment:
Do you really mean in parallel (if so - `[=in parallel=]`), or do you mean "in sequence"? I don't really think you want to resolve the Promise (3) in parallel with waiting for the event to have been fired (2).

Reply:
I meant [_in parallel_](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel) (in sequence on a different thread). Updated.